dual-gauge-card.js

```diff
@@ -258,6 +275,9 @@ class DualGaugeCard extends HTMLElement {
         --value-font-size: calc(var(--gauge-card-width) / 17);
         --title-font-size: calc(var(--gauge-card-width) / 14);
         --label-font-size: calc(var(--gauge-card-width) / 20);
+        
+        --outer-font-size: var(--outer-font-size); 
+        --inner-font-size: var(--inner-font-size);
```

Comment:
This sets those variables to themselves, which doesn't do much.

Instead the default value should be set here and should be relative to `--gauge-card-width`, like the other font sizes.
```suggestion
        --outer-font-size: calc(var(--gauge-card-width) / 15);
        --inner-font-size: calc(var(--gauge-card-width) / 15);
```